py-polars/tests/unit/datatypes/test_decimal.py

```diff
@@ -16,7 +16,7 @@ def permutations_int_dec_none() -> list[tuple[D | int | None, ...]]:
                 D("-0.01"),
                 D("1.2345678"),
                 D("500"),
-                -1,
+                D(-1),
```

Comment:
irrelevant to  partial schema. but test [`test_series_from_pydecimal_and_ints` ](https://github.com/pola-rs/polars/blob/d3721f15167c080df6c843bb6d8b07105e1fc932/py-polars/tests/unit/datatypes/test_decimal.py#L26) failed.